### PR TITLE
update administrate-field-image to 1.1.0

### PR DESCRIPTION
**Pull request by BenMorganIO**
This also caused bundler to update our Rails versions as well from 4.2.7.1 to 4.2.8.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -17,45 +17,43 @@ PATH
...
-      activemodel (= 4.2.7.1)
-      activesupport (= 4.2.7.1)
+    activerecord (4.2.8)
+      activemodel (= 4.2.8)
+      activesupport (= 4.2.8)
       arel (~> 6.0)
-    activesupport (4.2.7.1)
+    activesupport (4.2.8)
       i18n (~> 0.7)
-      json (~> 1.7, >= 1.7.7)
```
Comment (BenMorganIO) [inline Gemfile.lock]: yesssssssssss

File: Gemfile.lock [hunk 2]
```diff
@@ -17,45 +17,43 @@ PATH
...
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
     addressable (2.3.8)
-    administrate-field-image (1.0.0)
+    administrate-field-image (1.1.0)
       administrate (>= 0.2.0.rc1)
-      rails (>= 4.2)
```
Comment (BenMorganIO) [inline Gemfile.lock]: YAYYYYY one less spot for Rails to be set.

File: Gemfile.lock (hunk 1 above)
Comment (nickcharlton) [inline Gemfile.lock]: Oh wow. <3

File: Gemfile.lock (hunk 2 above)
Comment (nickcharlton) [inline Gemfile.lock]: Oh wow. <3

**Pull request merged**